.gitignore

```diff
@@ -136,6 +136,7 @@ publish/
 
 src/packages/
 !src/packages/repositories.config
+src/SampleLinter/packages
```

Comment:
Looks like all the other samples are putting their package dependencies in src/packages. Let's do the same, it also makes it a little easier for us to track third parties.